pills/07-working-derivation.xml

```diff
@@ -99,7 +99,7 @@
       <xi:include href="./07/simple-derivation.xml" />
 
       We did it! The contents of
-      <filename>/nix/store/w024zci0x1hh1wj6gjq0jagkc1sgrf5r-<emphasis>foo</emphasis></filename>
+      <filename>/nix/store/gczb4qrag22harvv693wwnflqy7lx5pb-<emphasis>foo</emphasis></filename>
```

Comment:
this will change whenever stdenv changes (several times a year). can probably not change this

Reply:
What about the other two changes. Are they OK to be committed ?